File: 7KYU/Sum of a sequence/solution.cs
using System;
public static class Kata
{
  public static int SequenceSum(int start, int end, int step)
  {
    if(start > end) return 0;
    var n = ((end - start) / step) + 1;
    return (n * ((2*start) + ((n - 1) * step)))/2;
  }
}

File: 7KYU/Sum of a sequence/solution.rs
fn sequence_sum(start: u32, end: u32, step: u32) -> u32 {
    let mut res: u32 = 0;
    let mut i:u32 = start;
    
    while i <= end {
        res = res + i;
        i = i + step;
    }
    
    return res;
}
